report/src/sections/01-introduction.tex

```diff
@@ -64,26 +64,26 @@ \subsection{Related Work}\label{subsec:related-work}
 
 
 \subsection{Contributions}\label{subsec:contributions}
-As described in \autoref{subsec:related-work}, one of our main contributions is describing an analysis made by extending the work made by Cortesi and Halder.
+As described in \autoref{subsec:related-work}, one of our main contributions is describing a static analysis made by extending the work made by Cortesi and Halder~\cite{halder_abstract_2012}.
 
 Included in this, we extend the level of abstraction used in the analysis.
 Where~\cite{halder_abstract_2012} only abstract over values, we abstract over both the tuples and values, and the multiplicity of the tuples and values in the database, ultimately guaranteeing a finite analysis.
 This is thoroughly covered in \autoref{sec:theory}.
 
-Also included in the contribution is a description of an analysis used to check properties when given a program and its associated database.
+Also included in the contribution is a description of a static analysis used to check properties when given a program and its associated database.
 While we have not provided a specific method for checking general properties, we have outlined a systematic approach for verifying invariants.
 Invariants are a particular type of property that should always be held during the program's execution.
 
-This way, we can run the analysis on the program and afterward check at each step of the program if the invariant holds.
+This way, we can analysis the program and afterward check if for all possible steps of the program the invariant holds.
 If it does not, we can stop the analysis and report that the program is incorrect.
 We can check for invariants in every state of the program.
 We do this in a program graph, where each node represents a state of the program.
 We can check if the invariant holds in each node.
 This way, we can systematically verify the program's correctness throughout its execution.
 
-The invariants can be checked with the predicate syntax which is described in \autoref{fig:abstract-syntax}.
+The invariants can be checked with the syntax which is described in \autoref{fig:abstract-syntax}.
```

Comment:
Please be more specific. Here we are refering to the to syntax of boolean formulars. Additionally you should be able to specify the state in which you want to check the invariant.